src/App.js

```diff
@@ -1,32 +1,29 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css'
 import ChartVis from './ChartVis'
 
 class App extends Component {
-    constructor(props) {
-      super(props)
-      this.state = {
-        data: []
-      }
-    
-    
+  constructor (props) {
+    super(props)
+    this.state = {
+      data: []
```

Comment:
This state items does not appear to be used? If it's not used yet, let's not put it in. Don't write code you don't have to write! 😄

Reply:
OK, I removed it. Yet I'm not clear on the constructor. After talking with Alex last night, I understood that this (data: []) would be used to bind data that would later be called in ComponentWillMount. Maybe you can clarify this for me--aside from the PR.